chain-registry/src/utils.rs

```diff
@@ -0,0 +1,62 @@
+use crate::error::RegistryError;
+use async_trait::async_trait;
+use http::uri::Builder;
+use serde::de::DeserializeOwned;
+
+pub const PROTOCOL: &str = "https";
+pub const HOST: &str = "raw.githubusercontent.com";
+pub const REGISTRY_PATH: &str = "/cosmos/chain-registry";
+pub const REF: &str = "ffdfbff3a21c7d2dbaac6f1e4730f47063f4bd72";
+
+pub const TEST_CHAINS: &[&str] = &["cosmoshub", "juno", "evmos", "osmosis", "regen"];
+
+/// `Fetchable` represents the basic expectations for external data or resources that
+/// can be fetched.
+#[async_trait]
+pub trait Fetchable
+where
+    Self: DeserializeOwned,
+{
+    /// The file name of the fetchable resource.
+    fn file_name() -> String;
```

Comment:
File names and paths are typically represented as a `PathBuf`:

```suggestion
    fn file_name() -> PathBuf;
```